Docs: add a try Tetragon on Linux getting-started tutorial

mtardy opened the pull request:
Preview [here](https://deploy-preview-899--tetragon.netlify.app/docs/getting-started/try-tetragon-linux/)

The goal is to make people discover Tetragon features quickly and easily. Avoiding getting too technical or into the details.

tixxdz reviewed (approved):
lgtm after renaming container tetragon-container and thank you ;-)

kkourt commented on docs/content/en/docs/getting-started/try-tetragon-linux.md:
```diff
@@ -0,0 +1,406 @@
+programs, `ls`, `uname` and `whoami`, and finally the exit of `sh`.
+
+{{< note >}}
+Since Tetragon is running directly on the host, you may observe a lot of
+unrelated from various process executions on your machine unrelated to what you
+typed during the shell session. Indeed, Tetragon is monitoring every process
```
```suggestion
Since Tetragon monitors all processes on the host  host, you may observe events
unrelated to what was
typed in the shell session. Indeed, Tetragon is monitoring every process
```

kkourt commented on docs/content/en/docs/getting-started/try-tetragon-linux.md:
```diff
@@ -0,0 +1,406 @@
+
+## Write a TracingPolicy to extend Tetragon events
+
+Tetragon observes the process lifecycle by default, but for more advanced use
+cases, you can write your own custom TracingPolicy. A TracingPolicy is a
```
```suggestion
Tetragon by default observes only process lifecycles.  More advanced use-cases, 
require configuring Tetragon via TracingPolicies. A TracingPolicy is a
```

kkourt commented on docs/content/en/docs/getting-started/try-tetragon-linux.md:
```diff
@@ -0,0 +1,406 @@
+
+Tetragon observes the process lifecycle by default, but for more advanced use
+cases, you can write your own custom TracingPolicy. A TracingPolicy is a
+user-configurable Kubernetes custom resource (CR) that allows users to trace
+arbitrary events in the kernel and optionally define actions to take on a
+match, for enforcement for example.
```
```suggestion
user-configurable Kubernetes custom resource (CR) that allows users to access additional functionality,
such as tracing arbitrary events in the kernel and, optionally, define actions to take on a match.
These actions can be, for example, used to implement enforcement.
```

kkourt commented on docs/content/en/docs/getting-started/try-tetragon-linux.md:
````diff
@@ -0,0 +1,406 @@
+}
+```
+
+You can see that we collected information on the process itself, with for
+example, the uid, the binary name, and the start\_time. But we also collected
+information on the parent of this process, here the binary `/bin/bash`. This is
+the same kind of information we get from the base process lifecycle sensors.
+Most importantly, we can see that we hooked the function `__x64_sys_openat`
+(for an arm64 host it would be `__arm64_sys_openat`) and we also collected the
+argument's value.
````
```suggestion
You can see that Tetragon collected information on the process itself, with for
example, the uid, the binary name, and the start\_time, as well as on the process parent,
`/bin/bash` in this case. This is
the same kind of information we get from the base process lifecycle sensors.
Finally, we can see that we hooked the function `__x64_sys_openat`
(for an arm64 host it would be `__arm64_sys_openat`) and we also collected the
argument's value.
```

kkourt commented on docs/content/en/docs/getting-started/try-tetragon-linux.md:
```diff
@@ -0,0 +1,406 @@
+
+{{< warning >}}
+This example is just an illustration and should not be used in a production
+environment, filtering an argument for security can be difficult, especially
```
```suggestion
environment.  There are many challenges in building a production-grade policy, that go beyond the scope
of this guide. For example, filtering an argument for security can be difficult, especially
```

kkourt reviewed (approved):
Have some suggestions, but generally this looks great to me! 🚀

mtardy commented:
> lgtm after renaming container tetragon-container and thank you ;-)

What do you think of `tetracon` :D! I'm searching for a contraction so that it's fast and easy to type and not confused with `tetragon` itself.

mtardy commented:
Plans to merge that with Tetragon `v0.9.1` release because of some Dockerfile and tetra fixes.

mtardy commented:
I pushed some changes ([see here](https://github.com/cilium/tetragon/compare/c50837efa3dc2c87804bdd0949bfbe235f073aee..22f04423531369fdbc80394e5ab68a819af3810e)):
- I simplified the Tutorial thanks to the new encoder on tetra that "understands" `openat` args (and updated the output).
- added a note about Docker Desktop on macOS and an FAQ on how to run Tetragon on Mac computers.

mtardy commented:
I finally fell back to using `tetragon-ci:latest` instead of a proper release because I will need Tetragon `0.10` release for this tutorial to work correctly. I prefer to edit it later when the next release is created. [See the comparison for more details](https://github.com/cilium/tetragon/compare/22f04423531369fdbc80394e5ab68a819af3810e..2abb11ad8535a59822f778ae05df93e3ce45204e).

Using a proper release will avoid some potential issues of people in the future but I prefer to merge this now because I think this is a nice addition.

mtardy commented:
This PR will solve the rest of the work of https://github.com/cilium/tetragon/issues/872.
